examples/simple_model_example_1.yml

```diff
@@ -0,0 +1,4 @@
+sample:
+    model:
+        origin: guess based on preparation
+        stack: air | 10 ( Si 7 | Fe 7 ) | Si
```

Comment:
This is the way that refnx specifies things, i.e. the structure is always defined as fronting --> layers --> backing, where the fronting is the medium that the radiation is incident through. I could be wrong but refl1d thinks about it from sample point-of-view. i.e. Silicon--> layers --> air. Did we normalise on a way to handle this?

Reply:
According to Jochen the consensus was to use this convention. Which has the added advantage that a list with 1-layer per line has the correct order, too.